tests/ops/qubit/test_state_prep.py

```diff
@@ -14,47 +14,12 @@
 """
 Unit tests for the available qubit state preparation operations.
 """
-import itertools
-import re
 import pytest
-import functools
-import copy
-import numpy as np
-from numpy.linalg import multi_dot
-from scipy.stats import unitary_group
-from scipy.linalg import expm
-from pennylane import numpy as npp
```

Comment:
were these not being used at all?

Reply:
Yep.  I guess when I split the testing file up into smaller testing files, the imports just got copied over. I wonder how this got past code factor though.